scripts/update-release.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Input version should be X.Y.Z
```

Comment:
No support for X.Y.Z-rc1?

Reply:
Good point.

Reply:
@MikeSpreitzer do we need that for RC?  docs are not uploaded for RC and also the readme is not changed (as the RC is not the "latest version" we want to point to

Reply:
https://github.com/kubestellar/kubestellar/blob/main/docs/content/direct/release.md#step-by-step makes no exception for test releases. The editing of the README remark about latest release needs some nuanced handling.

Reply:
Done. -rcX is supported as well

Reply:
You should update the comment text to reflect this change.  Can be done separately.